test/test_flowruns.py

```diff
@@ -13,6 +15,19 @@
 GET_BY_ID_INPROGRESS_XML = "flow_runs_get_by_id_inprogress.xml"
 
 
+def server_response_error_factory(code: str, summary: str, detail: str) -> str:
```

Comment:
This is nice, can you put it in a helper file so other tests can use it?

Reply:
Moved it to _utils